x-pack/solutions/security/plugins/security_solution/server/endpoint/services/actions/mocks.ts

```diff
@@ -5,11 +5,10 @@
  * 2.0.
  */
 
-import type { estypes } from '@elastic/elasticsearch';
+import type { estypes, TransportRequestOptions } from '@elastic/elasticsearch';
 import type { ElasticsearchClientMock } from '@kbn/core/server/mocks';
 import { AGENT_ACTIONS_RESULTS_INDEX } from '@kbn/fleet-plugin/common';
 import { Readable } from 'stream';
-import type { TransportRequestOptions } from '@elastic/transport';
```

Comment:
this change is because yarn got confused and left the old `@elastic/transport` in the top level and the matching version under `@elastic/elasticsearch/node_modules`.
Since we don't depend directly on `@elastic/transport`, I think that it's best to import from the client's library (that re-exports the transport)